### ISSUE TITLE
Solution to Exercise 3.7.5

### ISSUE THREAD

**Issue by timothydobbins** [jrnold/r4ds-exercise-solutions#207]
The suggestion for the second graph in Ex 3.7.5 is not quite correct. It seems that `..prop..` calculates proportions of color within cut, so stacking these yields proportions that can exceed 1 (for example, the total proportion for "Ideal" is presented as ~2.8.

To present the plot to be consistent with that in Part 1 (so that the height of the 'Ideal' bar is 0.4), the following can be used:

```
ggplot(data = diamonds) + 
  geom_bar(aes(x=cut, y = ..count../sum(..count..), fill = color))
```
![image](https://user-images.githubusercontent.com/4936029/53631300-58767600-3c66-11e9-8ea1-c37d4f339e53.png)

There might be a simpler way, but I couldn't find it.

**Comment by ryanatanner** [comment]
I think the way the question is framed, and the two example plots are provided is confusing. When reading the question, I assume that adding `group = 1` to each of the aesthetic mappings would 'fix' the chart, but it seems to not fix the second one. In the solutions provided, adding `group = color` also does not 'fix' the chart, but gets something closer to what @timothydobbins has provided. Does this warrant an issue filed on the actual question, rather than the solution, or are we still missing something?

**Comment by nielsenmarkus11** [comment]
I created pull request https://github.com/jrnold/r4ds-exercise-solutions/pull/208 as the complete fix to this issue.

**Comment by jrnold** [comment]
>  Does this warrant an issue filed on the actual question, rather than the solution, or are we still missing something?

I think it is a confusing question.

**Comment by jrnold** [comment]
Thanks @nielsenmarkus11 for figuring this out!

**Comment by jrnold** [comment]
Closed with #208
